app/scripts/background.js

```diff
@@ -356,7 +355,7 @@ function setupController(initState, initLangCode, remoteSourcePort) {
     storeAsStream(controller.store),
     debounce(1000),
     storeTransformStream(versionifyData),
```

Comment:
Do we still need this step? The "versionifyData" step is intended not just to move the state under the `data` property. It is also where the cached version is set. But now that's being set in `local-store.js` instead.

Reply:
Nope! Removing now.

Reply:
removed [here](https://github.com/MetaMask/metamask-extension/pull/15978/commits/fdeb4399ca7665ee03bb979fceb5ec6c07abe5f2)